docs/software/webserver_stack.md

```diff
@@ -1,25 +1,137 @@
+
+![dietpi-webstack-lamp](../assets/images/dietpi-software-webstack-lamp.jpg)
+
+=== "Quick start"
+
+    **Access website:**
+
+    - url = http://192.168.0.100 (`http://<your.local.ip>` or `http://<your.host.name>`)
+    - local directory = /var/www
+
+    **Access phpinfo:**
+
+    - url = http://192.168.0.100/phpinfo.php
+
+    **Access memcache info:**
+
+    - APC/u = http://192.168.0.100/apc.php
+    - Opcache = http://192.168.0.100/opcache.php
+
+    **MySql / MariaDB Details:**
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+    - username = root
+    - password = dietpi
+
+=== "Secured access - HTTPS/SSL"
+
+    **Let's Encrypt** is highly recommended - [see here how to install](../../dietpi_tools/#lets-encrypt-ssl-nordvpn-support). This will automate the creation and setup of your free SSL cert.
+
+    ??? note "Alternative way: Manually enable HTTP/SSL by installing a self-signed SSL certificate"
+
+        !!! caution ""
+            This method is recommended only in case **Let's encrypt** is not a viable option. 
+    
+        **Step 1. Create key**
+
+        ``` bash
+        mkdir -p /etc/apache2/ssl
+        openssl req -x509 -nodes -days 1000 -newkey rsa:2048 -keyout /etc/apache2/ssl/apache.key -out /etc/apache2/ssl/apache.crt
+        ```
+
+        **Step 2. Enable SSL config and restart Apache**
+
+        ``` bash
+        chmod 600 /etc/apache2/ssl/*
+        cat << _EOF_ > /etc/apache2/sites-enabled/default-ssl.conf
+
+        <IfModule mod_ssl.c>
+            <VirtualHost _default_:443>
+                    ServerAdmin webmaster@localhost
+                    ServerName example.com:443
+                    DocumentRoot /var/www
+
+                    LogLevel error
+                    ErrorLog ${APACHE_LOG_DIR}/error.log
+                    #CustomLog ${APACHE_LOG_DIR}/access.log combined
+
+                    SSLEngine on
+
+                    SSLCertificateFile /etc/apache2/ssl/apache.crt
+                    SSLCertificateKeyFile /etc/apache2/ssl/apache.key
+
+                    <FilesMatch "\.(cgi|shtml|phtml|php)$">
+                                    SSLOptions +StdEnvVars
+                    </FilesMatch>
+                    <Directory /usr/lib/cgi-bin>
+                                    SSLOptions +StdEnvVars
+                    </Directory>
+
+                    BrowserMatch "MSIE [2-6]" \
+                                    nokeepalive ssl-unclean-shutdown \
+                                    downgrade-1.0 force-response-1.0
+                    BrowserMatch "MSIE [17-9]" ssl-unclean-shutdown
```

Comment:
Let's remove this part for ancient Internet Explorer (I did already on our forum), it contains a typo anyway as `[17-9]` should have been `[7-9]` of course 😄.

Reply:
Agree, I will do this.